relatedProjects.html

```diff
@@ -90,21 +90,21 @@ <h2 id="project_tagline">
 						<div class="flex-item centered-content">
 							<div class="h-centered-content">
 								<a
-									href="https://github.com/SAP/openui5-charting-custom-controls"
+									href="https://github.com/ui5experts/ui5-schemas"
 									target="_blank"
 									class="image-link part-container">
-									<img src="images/projects/Chart-JS-Logo.png" class="project-image"/>
+									<img src="images/projects/UI5-Schemas.png" class="project-image"/>
 								</a>
 							</div>
 							<div class="description-container">
 								<h4>
 									<a
-										href="https://github.com/SAP/openui5-charting-custom-controls"
+										href="https://github.com/ui5experts/ui5-schemas"
 										target="_blank">
-										OpenUI5 Custom Controls for charting
+										UI5 Schemas
 									</a>
 								</h4>
-								This project shows how to use C3.js and Chart.js within OpenUI5 projects.
+								UI5 Schemas allows you to develop SAPUI5/OpenUI5 XML at a maximum convenience. It downloads, upgrades and sets up SAPUI5/OpenUI5 XML schemas for a better development experience in your favorite IDE (if it is WebStorm ;).
```

Comment:
the text is a bit long, for the featured area, please shorten it to:
UI5 Schemas allows you to develop SAPUI5/OpenUI5 XML views at a maximum convenience by setting up XML "schemas."

In the table you can leave the long text